src/plugins/kbn_vislib_vis_types/index.js

```diff
@@ -2,10 +2,8 @@ module.exports = function (kibana) {
 
   return new kibana.Plugin({
 
-    uiExports: {
-      visTypes: [
-        'plugins/kbn_vislib_vis_types/kbn_vislib_vis_types'
-      ]
+    ui: {
+      visTypes: './kbn_vislib_vis_types'
```

Comment:
Oh crap, I almost missed something on this line that perhaps is more urgent than the extension discussion: This is exporting a single string rather than an array, but it's using the `visTypes` property - I assume you intended to use `visType` here?